Add macOS caution to AdGuardDNS Client docs (#215)
AdguardTeam/KnowledgeBaseDNS: docs/dns-client/overview.md

1. Download and unpack the `.tar.gz` or `.zip` archive from the [releases page][releases].

    :::caution

>     On macOS it's crucial for daemons installed globally to be owned by root (see the [`launchd` documentation][launchd-requirements]), so the `AdGuardDNSClient` executable must be placed into the `/Applications/` directory or its subdirectory.

Reviewer: ```suggestion
    On macOS, it's crucial that globally installed daemons are owned by root (see the [`launchd` documentation][launchd-requirements]), so the `AdGuardDNSClient` executable must be placed in the `/Applications/' directory or its subdirectory.
```
